DevEdu.API/DevEdu.Business.Tests/TestDataHelpers/GroupData.cs

```diff
@@ -9,9 +9,14 @@ public class GroupData
     {
         public const int ExpectedAffectedRows = 1;
         public const int GroupId = 1;
+        public const int LessonId = 1;
+        public const int TaskId = 1;
+        public const int UserId = 1;
+        public const int UserIdAuthorization = 1;
```

Comment:
please remove all these consts from GroupData
and place them as variables inside the "given" section of the corresponding test methods